### DISCUSSION TITLE
How to insert new lines in delimiter pair depending on the input?

### DISCUSSION THREAD

**Discussion by gaoqiangks** [kylechui/nvim-surround#248] [Q&A] [answered]
For example,  I have the following configuration.
`["E"] = {`
`	add = function()`
`        local config = require("nvim-surround.config")`
`        local result = config.get_input("Input:")`

I want to add new lines in delimeter pair when `result = "equation"`, and don't add new lines when `result = "$"`.  Is there any way to realize this?

**Comment by kylechui** [comment, marked as answer]
The main thing to note is that multi-line strings are denoted using tables of strings. Thus `{ "$", "" }, { "", "$" }` is a pair of dollar signs on new lines above/below the cursor. You should be able to use an `if` statement to determine what value you wish to return. More info can be found in the `surrounds.add` section of the help menu.
